runtime-management/runtime-management-api/src/main/java/io/cloudslang/runtime/api/python/PythonExecutorCommunicationService.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2014-2017 EntIT Software LLC, a Micro Focus company (L.P.)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.cloudslang.runtime.api.python;
+
+import io.cloudslang.runtime.api.python.entities.PythonExecutorDetails;
+
+import javax.ws.rs.core.Response;
+
+public interface PythonExecutorCommunicationService {
```

Comment:
This service should be more generic. We shouldn't design it for certain calls, but to perform any kind of request to python executor. Methods should have as input the request path, method type (POST/GET), payload if necessary. Consider having secured methods or not (e.g health doesn't need security).